direct/data/mri_transforms.py

```diff
@@ -832,6 +919,11 @@ def forward(self, sample: Dict[str, Any]) -> Dict[str, Any]:
             # Shape (batch, coil, height, width, complex=2)
             sensitivity_map = T.safe_divide(acs_image, acs_image_rss)
         else:
+            if sample[self.kspace_key].ndim > 5:
+                raise NotImplementedError(
+                    "EstimateSensitivityMapModule is not yet implemented for "
+                    "Espirit sensitivity map estimation for 3D data."
+                )
```

Comment:
Perhaps you can also add this to the documentation string.

Reply:
Done